netlib/http/request.py

```diff
@@ -6,7 +6,9 @@
 from six.moves import urllib
 
 from netlib import utils
-from netlib.http import cookies
+import netlib.http.url
```

Comment:
This is to avoid stepping on a number of naked occurrences of the "url" name in the file itself. Renaming those would be ok, but seemed more invasive than I want for this patch (which should be an almost 0 behaviour changed refactoring).